packages/presentation/lib/screen/login/widgets/login_text_field.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/config/dimens/dimens.dart';
+import 'package:presentation/config/theme/app_colors.dart';
+
+class LoginTextField extends StatefulWidget {
```

Comment:
rename AuthTextField or AppTextField

Reply:
I don't like this idea, because I name due to screen
It can't be AppTextField because it isn't use in whole project